front/src/modules/trainschedule/helpers/updateTimetableItemHelpers.ts

```diff
@@ -141,29 +142,29 @@ export async function storeTrainSchedule(
 }
 
 export async function storePacedTrain(
-  timetableItemIdToUpdate: TimetableItemId,
+  timetableItemIdToUpdate: TimetableItemToEditData,
```

Comment:
Why do you pass the TimetableItemToEditData here ? You only use the timetableItemId
```suggestion
  timetableItemIdToUpdate: TimetableItemId,
```

Reply:
You're right, done